src/main/java/com/atomshermental/springbootnewsservice/Implementations/NewsRepositoryImplementation.java

```diff
@@ -0,0 +1,36 @@
+package com.atomshermental.springbootnewsservice.Implementations;
+
+import com.atomshermental.springbootnewsservice.Interfaces.NewsRepository;
+import com.atomshermental.springbootnewsservice.Objects.News;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import org.springframework.context.annotation.Primary;
+import org.springframework.stereotype.Repository;
+
+import java.util.*;
+
+@RequiredArgsConstructor
+
+@Repository
+@Primary
+public class NewsRepositoryImplementation implements NewsRepository {
+
+    private final Map<Long,News> storage;
+    private Long max = 0L;
```

Comment:
max initial value in repository must be calculated based on an initial storage value